charts/node/templates/statefulset.yaml

```diff
@@ -394,7 +417,7 @@ spec:
           args:
             - -c
             - |
-              set -eu
+              set -eux
```

Comment:
Set the x conditionally as we do in init containers?

Reply:
Fixed, added option to disable/enable this behavior